### ISSUE TITLE
About Timezone

### ISSUE THREAD

**Issue by mchydhamlna** [hcgcloud/pterodactyl-wine-samp#1]
How to synchronize the timezone with the pterodactyl panel ? because this egg has a different timezone from the one on the panel

**Comment by mchydhamlna** [comment]
my timezone on panel Asia/Jakarta but on samp wine use UTC

**Comment by Dannntzy** [comment]
Yes, I also experienced the same thing, timezone to UTC, how do I change the timezone
